### PR TITLE
Update KubernetesPodOperator kubernetes_min_pod example

### PR DESCRIPTION
**Pull request by leahecole**
In this oprator, the kubernetes_min_pod fails without the 'cmds' parameter set

### CODE REVIEW

**Comment by leahecole** [comment]
@tswast , the Travis CI build is failing on the pythonvirtualenvoperator_python2_test.py - are these shells of tests that are in here? Any ideas of how to proceed?

**Comment by tswast** [comment]
@leahecole Yes, the Travis output is from the tests here.

```
nox > flake8 --show-source --builtin gettext --max-complexity 20 --import-order-style google --exclude .nox,.cache,env,lib,generated_pb2,*_pb2.py,*_pb2_grpc.py --ignore=E121,E123,E126,E226,E24,E704,W503,W504,I100,I201,I202 --application-import-names use_local_deps,pythonvirtualenvoperator_python2,unit_testing_variables,unit_testing,simple,unit_testing_test,bq_notify,connections,__init__,trigger_response_dag_test,quickstart_test,kubernetes_pod_operator_test,connections_test,bq_notify_test,bq_copy_across_locations,bashoperator_python2_test,kubernetes_pod_operator,simple_test,dependencies,use_local_deps_test,trigger_response_dag,bq_copy_across_locations_test,unit_testing_cycle,bashoperator_python2,quickstart,pythonvirtualenvoperator_python2_test .
./pythonvirtualenvoperator_python2_test.py:15:1: F401 'sys' imported but unused
import sys
^
./pythonvirtualenvoperator_python2_test.py:17:1: F401 'pytest' imported but unused
import pytest
^
nox > Command flake8 --show-source --builtin gettext --max-complexity 20 --import-order-style google --exclude .nox,.cache,env,lib,generated_pb2,*_pb2.py,*_pb2_grpc.py --ignore=E121,E123,E126,E226,E24,E704,W503,W504,I100,I201,I202 --application-import-names use_local_deps,pythonvirtualenvoperator_python2,unit_testing_variables,unit_testing,simple,unit_testing_test,bq_notify,connections,__init__,trigger_response_dag_test,quickstart_test,kubernetes_pod_operator_test,connections_test,bq_notify_test,bq_copy_across_locations,bashoperator_python2_test,kubernetes_pod_operator,simple_test,dependencies,use_local_deps_test,trigger_response_dag,bq_copy_across_locations_test,unit_testing_cycle,bashoperator_python2,quickstart,pythonvirtualenvoperator_python2_test . failed with exit code 1
nox > Session lint(sample='./composer/workflows') failed.
```

The `flake8` command found imports that aren't used. Remove them to pass the linter.

I recommend getting a Python development environment setup with multiple Python versions (see: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/MAC_SETUP.md) for how I have my system setup. Then see the instructions at https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#using-nox for running `nox` to run the tests locally.

In your case, run (in a Python 3 virtual environment) from the `python-docs-samples/` directory these command:

```
pip install --upgrade nox
nox -s "lint(sample='./composer/workflows')"
```

That should replicate what Travis is doing.

**Comment by leahecole** [comment]
@tswast sorry for the delay - I lost sight of this. Cool to merge? All I changed was commenting out the things that made the linter barf :)

File: composer/workflows/pythonvirtualenvoperator_python2_test.py [hunk 1]
```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import sys
+# import sys
```
Comment (tswast) [inline composer/workflows/pythonvirtualenvoperator_python2_test.py]: Delete these lines rather than comment them out.

**Pull request merged**